frontend/app/src/locales/en.json

```diff
@@ -57,6 +57,25 @@
     "subtitle": "Add, edit, remove tokens and assets. Changes in the assets will be reflected across all your accounts",
     "underlying_tokens": "Underlying Tokens"
   },
+  "asset_update": {
+    "buttons": {
+      "skip": "Skip",
+      "update":"Update"
+    },
+    "description": "Version {remote} of the asset database is available. Would you like to update from version {local}?",
+    "manual": {
+      "check": "Check For Update",
+      "skipped": "Update notifications are disabled for version {skipped}",
+      "subtitle": "Perform a check for an updated version of the asset database",
+      "title": "Check for asset db updates"
+    },
+    "skip_notification": "Don't notify me until the next update",
+    "success": {
+      "description": "The asset database has been successfully updated to version {remoteVersion}",
```

Comment:
The remote version here does not seem to be replaced in the text. Not sure exctly how it works. But judging from the other entries using `{remote}` instead would work?

![2021-03-26_14-43](https://user-images.githubusercontent.com/1658405/112640745-e9b67780-8e41-11eb-97d4-47f294fd0025.png)

Reply:
I forgot to pass the value to the template. It should be fine now